src/commands/Fun Commands/MemeCommand.ts

```diff
@@ -3,7 +3,8 @@ import { Util } from 'discord.js';
 import * as ImageAPI from 'imageapi.js';
 
 export const run: RunFunction = async (client, message) => {
-	const meme = await ImageAPI.advanced('meme');
+	let memeSubreddits = ['meme', 'memes', 'dankmemes']
```

Comment:
use const over let

Reply:
This doesn't need to be changed, as the array is never going to be modified.